public/pages/workflow_detail/workflow_inputs/workflow_inputs.tsx

```diff
@@ -676,6 +675,7 @@ export function WorkflowInputs(props: WorkflowInputsProps) {
                           onClick={() => {
                             validateAndRunIngestion();
                           }}
+                          // TODO: only enable if ingest is dirty
```

Comment:
What do we mean here by "dirty"?

Reply:
form is dirty / unsaved changes

Reply:
Can we have a better name here? `Unsaved` or something

Reply:
It's a common term for unsaved form data, and is used throughout `Formik` form library we use to maintain form and state. Prefer to leave as-is for consistency.

Reply:
I see. Sounds good